src/actions/contactsActions.js

```diff
@@ -87,26 +86,9 @@ export const syncContactAction = (userId: string) => {
     const {
       user: { data: { walletId } },
       contacts: { data: contacts },
-      accessTokens: { data: accessTokens },
-      connectionIdentityKeys: { data: connections },
     } = getState();
 
-    let userInfo;
-    const connection = connections.find((conn: ConnectionIdentityKey) => conn.targetUserId === userId);
-    if (connection) {
-      userInfo = {
-        ...connection.targetUserInfo,
-        id: connection.targetUserId,
-      };
-    } else {
-      const accessToken = accessTokens.find(token => token.userId === userId);
-      if (!accessToken) return;
-      userInfo = await api.userInfoById(userId, {
-        walletId,
-        userAccessKey: accessToken.myAccessToken,
-        targetUserAccessKey: accessToken.userAccessToken,
-      });
-    }
+    const userInfo = await api.userInfoById(userId, walletId);
```

Comment:
`userInfoById` requires objet payload:
```suggestion
    const userInfo = await api.userInfoById(userId, { walletId });
```